### PR TITLE
Prepare 2.0.8 patch

### PR DESCRIPTION
**Pull request by natemcmaster**
Begins work for 2.0.8. Includes version bumps for most repos, but not all (see build/submodules.props).

### CODE REVIEW

**Comment by natemcmaster** [comment]
Merging to get the build moving along. If you find something in review, let me know.

File: build/dependencies.props [hunk 1]
```diff
@@ -13,6 +13,7 @@
     <MicrosoftAspNetIdentityEntityFrameworkPackageVersion>2.2.1</MicrosoftAspNetIdentityEntityFrameworkPackageVersion>
     <MicrosoftAspNetWebApiClientPackageVersion>5.2.2</MicrosoftAspNetWebApiClientPackageVersion>
     <MicrosoftAzureKeyVaultPackageVersion>2.3.2</MicrosoftAzureKeyVaultPackageVersion>
+    <MicrosoftAzureServicesAppAuthenticationPackageVersion>1.1.0-preview</MicrosoftAzureServicesAppAuthenticationPackageVersion>
```
Comment (JunTaoLuo) [inline build/dependencies.props]: Why is this now no longer an external dependency?

File: build/dependencies.props [hunk 2]
```diff
@@ -32,7 +33,7 @@
     <MicrosoftNETCoreApp11PackageVersion>1.1.5</MicrosoftNETCoreApp11PackageVersion>
     <MicrosoftNETCoreApp20PackageVersion>2.0.7</MicrosoftNETCoreApp20PackageVersion>
     <MicrosoftNETCoreWindowsApiSetsPackageVersion>1.0.1</MicrosoftNETCoreWindowsApiSetsPackageVersion>
-    <MicrosoftNETTestSdkPackageVersion>15.6.1</MicrosoftNETTestSdkPackageVersion>
+    <MicrosoftNETTestSdkPackageVersion>15.3.0</MicrosoftNETTestSdkPackageVersion>
```
Comment (JunTaoLuo) [inline build/dependencies.props]: Why the downgrade?

Comment (natemcmaster) [reply build/dependencies.props]: dotnet-test fails with reasons I haven't had time to investigate with the new version. There wasn't any really reason to upgrade from 15.3, so reverted it to a working version.

File: build/dependencies.props (hunk 1 above)
Comment (natemcmaster) [reply build/dependencies.props]: Microsoft.Azure.Services.AppAuthentication is a new external dependency of a package shipping in 2.0.8 for the first time, Microsoft.AspNetCore.AzureKeyVault.HostingStartup.

cc @pakrym

Comment (JunTaoLuo) [reply build/dependencies.props]: Ah I see

File: build/dependencies.props (hunk 2 above)
Comment (JunTaoLuo) [reply build/dependencies.props]: Sounds suspicious but okay.

**Pull request merged**